src/org/plovr/ConfigOption.java

```diff
@@ -272,6 +272,16 @@ public void apply(String outputCharset, Config.Builder builder) {
     }
   }),
 
+
+  CACHE_OUTPUT_FILE("cache-output-file", new ConfigUpdater() {
```

Comment:
is there any reason not to make this behavior the default? Presumably plovr could just pick a tmpdir itself.